pkg/config/config.go

```diff
@@ -585,6 +586,8 @@ func (m *Manager) setDefaults(prefix string, flags *pflag.FlagSet, config interf
 				}
 				flags.StringP(name, shorthand, def, description)
 
+			case []ratelimit.Profile:
```

Comment:
Why is this necessary? Can't we just tell the config package to ignore the config with `name:"-"`?

Reply:
The `[]ratelimit.Profile` is user configuration, but only accepted from a config file and not from command line arguments.